api/runner/task.go

```diff
@@ -2,10 +2,12 @@ package runner
 
```

Comment:
Shouldn't this code be at iron-io/runner rather than in functions? All this code is driver-specific.